test/integration/parallel.spec.js

```diff
@@ -18,4 +18,42 @@ describe('parallel run', () => {
     assert.strictEqual(result.stats.failures, 1);
     assert.strictEqual(result.stats.passes, 2);
   });
+
+  it('should correctly set worker ids for each process', async () => {
+    const result = await runMochaJSONAsync('parallel/testworkerid3.mjs', [
+      '--parallel',
+      '--jobs',
+      '2',
+      require.resolve('./fixtures/parallel/testworkerid1.mjs'),
+      require.resolve('./fixtures/parallel/testworkerid2.mjs')
+    ]);
+    // test 1 always runs on worker 0
+    assert.strictEqual(
```

Comment:
Does GH always allocates enough processes (two) to keep this test stable?

Reply:
Yes, --jobs 2 requests `workerpool` to create 2 processes, no matter how many CPUs are available.